python/IECoreHoudini/FnParameterisedHolder.py

```diff
@@ -88,7 +88,10 @@ def _doCreate( nodeType, name, className, version=None, envVarName=None, parent=
 		if contextArgs.get( "shiftclick", False ) :
 			converter = holder.parent().createNode( "ieCortexConverter", node_name = holder.name()+"Converter" )
 			outputNode = hou.node( contextArgs.get( "outputnodename", "" ) )
-			toolutils.connectInputsAndOutputs( converter, False, holder, outputNode, 0, 0 )
+			try:
```

Comment:
I'd rather not use `try/except` as it will mask any errors in the called function. Just check the version like you had been doing please.